src/test/java/net/revelc/code/impsort/ImpSortTest.java

```diff
@@ -237,5 +237,31 @@ public void testJava14PreviewFeatures() throws IOException {
         lines.stream().filter(line -> line.contains("public record Java14Preview")).count());
   }
 
+  @Test
+  public void testJava21RecordDeconstruction() throws IOException {
+    Path p = Paths.get(System.getProperty("user.dir"), "src", "test", "resources",
```

Comment:
The test should verify that a problem was actually seen, but ignored, in order to make sure that the feature is working properly and we didn't just get lucky that it parsed.

Reply:
Yes, indeed. Now checking the problems.